src/main/java/duke/finalObjects/TootieErrorMsgs.java

```diff
@@ -0,0 +1,23 @@
+package duke.finalObjects;
+
+public class TootieErrorMsgs {
+    public static final String NEWLINE = System.lineSeparator();
+
+    public static final String TasklistEmptyMsg = "No tasks found! " + TootieSymbols.HAPPY_EMOTICON;
+    public static final String TODO_WRONG_FORMAT_MSG = "Check todo input formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.TODO_COMMAND_DESCRIPTION;
+    public static final String DEADLINE_WRONG_FORMAT_MSG = "Check deadline input formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.DEADLINE_COMMAND_DESCRIPTION;
+    public static final String DUEDATE_WRONG_FORMAT_MSG = "Check due date formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.DEADLINE_COMMAND_DESCRIPTION + NEWLINE + NEWLINE + TootieNormalMsgs.DATE_FORMAT_MESSAGE + NEWLINE;
+    public static final String EVENT_WRONG_FORMAT_MSG = "Check event input formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.EVENT_COMMAND_DESCRIPTION;
+    public static final String STARTDATE_WRONG_FORMAT_MSG = "Check start date formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.EVENT_COMMAND_DESCRIPTION + NEWLINE + NEWLINE + TootieNormalMsgs.DATE_FORMAT_MESSAGE + NEWLINE;
+    public static final String ENDDATE_WRONG_FORMAT_MSG = "Check end date formatting!" + NEWLINE + NEWLINE + TootieNormalMsgs.EVENT_COMMAND_DESCRIPTION + NEWLINE + NEWLINE + TootieNormalMsgs.DATE_FORMAT_MESSAGE + NEWLINE;
```

Comment:
good job including the correct format at the end of the error msg, greatly improves user experience and user friendliness